.env.example

```diff
@@ -32,6 +32,7 @@ FILE_API_KEY_NEXUS=👻
 FILE_API_KEY_BARNET=👻
 FILE_API_KEY_LAMBETH=👻
 FILE_API_KEY_SOUTHWARK=👻
+FILE_API_KEY_EPSOM_EWELL=👻
```

Comment:
I've added a card to the blue trello to set and control these values via the DB in much the same way we do other team integration values.

https://trello.com/c/A8gB8rSx/3177-configure-api-keys-for-file-download-via-teamintegrations-table